src/serlo-editor/plugins/article/editor.tsx

```diff
@@ -22,8 +22,18 @@ export function ArticleEditor({ editable, state }: ArticleProps) {
 
   const modalStrings = useEditorStrings().templatePlugins.article.addModal
 
+  // Focus the article introduction explanation on page load
+  // TODO: Move this into a focus-dedicated hook
+  const ref = useRef<HTMLDivElement | null>(null)
+  useEffect(() => {
+    const explanationTextPlugin = ref.current?.querySelector<HTMLDivElement>(
+      '.explanation-wrapper > .plugin-wrapper-container'
+    )
+    explanationTextPlugin?.focus()
```

Comment:
since you have a comment already this would also work:
```suggestion
    ref.current?.querySelector<HTMLDivElement>(
      '.explanation-wrapper > .plugin-wrapper-container'
    )?.focus()
```

Reply:
Did that already, and then moved it back to the current state, because a dangling property/method accessor is hard to detect. This is more code, but makes sure that the `?.focus` part doesn't get overlooked.